db/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
 --
 
--- Dumped from database version 9.5.6
--- Dumped by pg_dump version 9.5.6
+-- Dumped from database version 9.5.4
+-- Dumped by pg_dump version 9.5.4
```

Comment:
Isn't that what I said? Was there a reason you felt the need to repeat my comment?